interview-questions.md

```diff
@@ -7,44 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating an API with a model called Animal that has_many Sightings, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: The foreign key would be part of the Animal model and the name would be called animal_id. In order to fix this, you can go into the console and add a new migration.
 
-  Researched answer:
+  Researched answer: There must be a row already existing in order to add the foreign key.
```

Comment:
What happens when I am updating my DB and I want to associate different tables? what tool do I use?